backend/apps/web/routers/utils.py

```diff
@@ -109,16 +108,15 @@ async def download(
 
 @router.post("/upload")
 def upload(file: UploadFile = File(...)):
-    os.makedirs("./data/uploads", exist_ok=True)
-    file_path = os.path.join("./data/uploads", file.filename)
+    file_path = f"{UPLOAD_DIR}/{file.filename}"
 
     # Save file in chunks
-    with open(file_path, "wb+") as f:
+    with file_path.open("wb+") as f:
```

Comment:
Any reason for this change?

Reply:
fixed

Reply:
The change seem to have been introduced in [5b26d2a](https://github.com/ollama-webui/ollama-webui/pull/536/commits/5b26d2a686d47822d5959e90a1d0b64ebec88d67) :)